test/distributions/test_distributions.py

```diff
@@ -5389,6 +5389,9 @@ def __init__(self, probs):
         )
         laplace = pairwise(Laplace, [-2.0, 4.0, -3.0, 6.0], [1.0, 2.5, 1.0, 2.5])
         lognormal = pairwise(LogNormal, [-2.0, 2.0, -3.0, 3.0], [1.0, 2.0, 1.0, 2.0])
+        multinomial = tuple(
```

Comment:
I'm not actually thrilled with this test setup, but I couldn't use `pairwise` since `Multinomial` takes the `total_counts` argument before `probs`. I thought of trying to use `partial` or maybe a wrapper class that switches the order of the kwargs for testing purposes, but felt this was cleanest.